app/script/media/MediaStreamHandler.js

```diff
@@ -194,7 +194,25 @@ z.media.MediaStreamHandler = class MediaStreamHandler {
    * @returns {Promise} Resolves when the MediaStream has been replaced
    */
   replaceMediaStream(mediaStreamInfo) {
-    const {stream: mediaStream, type} = mediaStreamInfo;
+    const {stream: mediaStream} = mediaStreamInfo;
+
+    const replaceMediaStream = (localMediaStream, newMediaStreamInfo) => {
```

Comment:
The naming is confusing. This is in the replaceMediaStream. We do a replace when we update the joined call. This is more like an update local media stream functionality.

Reply:
Quick suggestion: How about we rename the method `updateMediaStream` and depending on the case it can either `replaceMediaStream` or `replaceMediaTrack` so that we have the same naming for both subfunctions?
What do you think?